### PR TITLE
[4.1] Unification of the language com_scheduler, part 3

### PR DESCRIPTION
**Pull request by tecpromotion**
### Summary of Changes

During the translation into German and testing, an error occurred in the display of the duration time.

This PR shows the correct output.

### Testing Instructions

create a new taks an click Run Test.

### Documentation Changes Required

.

### CODE REVIEW

**Comment by brianteeman** [comment]
ouch - I made that change and none of the testers spotted it

**Comment by tecpromotion** [comment]
> ouch - I made that change and none of the testers spotted it

yes, unfortunately i didn't test that enough with my team when beta1 was released. but we should test it thoroughly again before the stable release.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 8474a84ccf820579dbb3f20014758fdaf0278a8d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36297">issues.joomla.org/tracker/joomla-cms/36297</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36297">issues.joomla.org/tracker/joomla-cms/36297</a>.</sub>

File: administrator/language/en-GB/com_scheduler.ini [hunk 1]
```diff
@@ -132,7 +132,7 @@ COM_SCHEDULER_TASK_TYPE="Task Type"
 COM_SCHEDULER_TASK_TYPE_ASC="Task Type Ascending"
 COM_SCHEDULER_TASK_TYPE_DESC="Task Type Descending"
 COM_SCHEDULER_TEST_RUN="Run Test"
-COM_SCHEDULER_TEST_RUN_DURATION="Duration: %d seconds"
+COM_SCHEDULER_TEST_RUN_DURATION="Duration: %s seconds"
```
Comment (alikon) [inline administrator/language/en-GB/com_scheduler.ini:134]: it seems wrong to me `%s`  it should be `%d` - Signed decimal number (negative, zero or positive) we are talking of seconds

**Review by alikon — CHANGES_REQUESTED** [review]

File: administrator/language/en-GB/com_scheduler.ini (hunk 1 above)
Comment (wojsmol) [reply administrator/language/en-GB/com_scheduler.ini:134]: @alikon Please see screenshots in PR description.

**Comment by khu5h1** [comment]
I have tested this item :white_check_mark: successfully on 8474a84ccf820579dbb3f20014758fdaf0278a8d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36297">issues.joomla.org/tracker/joomla-cms/36297</a>.</sub>

File: administrator/language/en-GB/com_scheduler.ini (hunk 1 above)
Comment (bembelimen) [reply administrator/language/en-GB/com_scheduler.ini:134]: Probably it should be ```%f```?

**Comment by Shubhamverma2796** [comment]
I have tested this item :white_check_mark: successfully on 8474a84ccf820579dbb3f20014758fdaf0278a8d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36297">issues.joomla.org/tracker/joomla-cms/36297</a>.</sub>

File: administrator/language/en-GB/com_scheduler.ini (hunk 1 above)
Comment (wojsmol) [reply administrator/language/en-GB/com_scheduler.ini:134]: @bembelimen `%f` should be ok.

**Comment by wojsmol** [comment]
@tecpromotion Please use `%f` instead.

**Comment by richard67** [comment]
@tecpromotion Could you have a look on the review suggestions and check if "%f" works, and if so, change to that? I'd like to come forward here, but I'm a bit reluctant to set RTC as long as there are open review discussions.

**Comment by tecpromotion** [comment]
> @tecpromotion Could you have a look on the review suggestions and check if "%f" works, and if so, change to that? I'd like to come forward here, but I'm a bit reluctant to set RTC as long as there are open review discussions.

I can check this later.

**Comment by tecpromotion** [comment]
> @tecpromotion Please use `%f` instead.

@wojsmol Sorry but '%f' does not work here.

https://github.com/joomla/joomla-cms/blob/b76ec5536e2719fa46d89a4c4363c9b185d9fd39/build/media_source/com_scheduler/js/admin-view-run-test-task.es6.js#L60
Here a '%s' is searched for and replaced.


Change both to %f does not work.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36297">issues.joomla.org/tracker/joomla-cms/36297</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by alikon** [comment]
merged ? !!!!
why ? :face_with_head_bandage: 
evidently feedback from some are not welcomed or better simply ignored

**Comment by richard67** [comment]
@alikon Why ignored? @tecpromotion has checked with "%s" as requested and has reported back why we can't do that with the js as it is now.

**Comment by alikon** [comment]
this doesn't prove it's correct

**Comment by tecpromotion** [comment]
> this doesn't prove it's correct

You're absolutely right, but that doesn't solve the problem.
A developer has to sit down and fix it everywhere.
Now there is an output and no more error.

**Pull request merged**